stylus/cozy-ui/defaults.styl

```diff
@@ -102,6 +102,9 @@ body
     &.coz-btn-alert--success
         @extend $button-alert--success
 
+:global .coz-btn--upload-menu-item
```

Comment:
Those classes are already in a `:global`, you actually don't need it but you do need the `&`